public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -182,17 +182,10 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel,
   const uid = oldModel.uid;
   const serializerVersion = config.featureToggles.dashboardNewLayouts ? 'v2' : 'v1';
 
-  if (oldModel.templating?.list?.length) {
```

Comment:
We still need this check, otherwise it ends up being undefined, which `createVariablesForDashboard` doens't like

Reply:
I update the single failing test for this.